public/locales/cs/translation.json

```diff
@@ -20,6 +20,14 @@
     "namePlaceholder": "Název systému",
     "create": "Vytvořit"
   },
+  "deleteSystemModal": {
+    "title": "Smazat systém",
+    "explanation": "Smazáním systému budou smazány i všechny komponenty, funkce a režimy poruch. Pokračovat ve smazání systému?"
```

Comment:
@grotskat 
Does it makes sense that `režimy poruch` = `failure modes` ?

Reply:
@blcham 
"způsoby selhání"

Reply:
@kostobog Fixed in https://github.com/kbss-cvut/fta-fmea-ui/commit/50bb1183b9f6c58dd1b7d4c1bce6e54ef3f1748a, hope it is ok.